packages/w3up-client/test/capability/access.test.js

```diff
@@ -33,6 +33,7 @@ describe('AccessClient', () => {
       })
 
       const alice = new Client(await AgentData.create(), {
+        // @ts-ignore
```

Comment:
I may be able to get rid of these ts-ignore (remember: the whole file was ignored by tsc before) in a followup PR by removing the `voucher` type prop from `@web3-storage/access/types` ServiceType.